### ISSUE TITLE
Security issue in V10.13.0

### ISSUE THREAD

**Issue by h3mmy** [traefik/traefik-helm-chart#555]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik-helm-chart/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What version of the Traefik's Helm Chart are you using?

10.11.2

### What version of Traefik are you using?

2.6.0

### What did you do?

Scanned for security vulnerabilities

### What did you see instead?

A High Severity CVE

### What is your environment & configuration?

Kubernetes, YAML, etc

### Additional Information

```shell
I'm not sure that I should be adding details on a security issue but it's fairly straightforward to detect. github.com/hashicorp/consul 1.10.3 is the dependency at risk. My request is to update it to 1.10.4 where it is fixed.
```

**Comment by ddtmachado** [comment]
Hello @h3mmy,

Thanks for the report. Sometimes a security issue found in a dependency does not affect the core product for a lack of surface contact or even usage of the affected code. 

Can you be more specific on the CVE in question?

Also if this turns out to be a real security issue that needs fixing it should be addressed on the [Traefik repo](https://github.com/traefik/traefik) instead.

**Comment by h3mmy** [comment]
Of course! 

Package: github.com/hashicorp/consul
Installed Version: v1.10.3
Vulnerability CVE-2021-41805
Severity: HIGH
Fixed Version: 1.8.17, 1.9.11, 1.10.4
Link: [CVE-2021-41805](https://avd.aquasec.com/nvd/cve-2021-41805)

My standard practice is to avoid specifying the vulnerability without permission. I can open an issue with the traefik base repo if you think that's a better place for it. This could also be an opportunity for consideration of dependency override capabilities in the helm values. 

Thanks for all your work!

**Comment by ddtmachado** [comment]
Thank you for the specifics, on this one we are indeed not affected since we do not create or edit namespaces, this is more of a consul server security issues and not o client side one as noted on the consul changelog itself:

> namespaces: (Enterprise only) Creating or editing namespaces that include default ACL policies or ACL roles now requires acl:write permission in the default namespace. This change fixes CVE-2021-41805.

So it looks like a false positive from the scanning tool, which can happen from time to time.
